chat-commands.js

```diff
@@ -1855,23 +1852,22 @@ exports.commands = {
 		}
 		let reason = '';
 		if (targetUser && targetUser.locked && targetUser.locked.charAt(0) === '#') {
-			reason = ' (' + targetUser.locked + ')';
+			reason = ` (${targetUser.locked})`;
 		}
 
 		let unlocked = Punishments.unlock(target);
 
 		if (unlocked) {
-			const unlockMessage = unlocked.join(", ") + " " + ((unlocked.length > 1) ? "were" : "was") +
-				" unlocked by " + user.name + "." + reason;
+			const unlockMessage = `${unlocked.join(", ")} ${((unlocked.length > 1) ? "were" : "was")} unlocked by ${user.name}. ${reason}`;
```

Comment:
``reason`` comes with a space, if there is a reason; you're adding an extra here.